docs/FAQ.md

```diff
@@ -275,3 +275,60 @@ oci-onm-logan:
       worker: 1
 ```       
 
+### Log Collection for OCNE (Oracle Cloud Native Environment)
```

Comment:
On all troubleshoot related faqs pls add the location of files/etc where users might see these error messages.